### PR TITLE
Complete send message to attendees method

### PR DESCRIPTION
**Pull request by mahimabhayana**

### CODE REVIEW

File: scripts/socialBot.coffee [hunk 1]
```diff
@@ -100,6 +102,24 @@ parseNotifyUsers = (event) ->
 parseCreators = (event) ->
   return event.creators.join(', ')
 
+notifyAllAttendees = (res) ->
+  eventName = res.match[1].trim()
+  message = res.match[2].trim()
+  user = res.message.user.name
+  events = getEvents(res.robot.brain)
```
Comment (lmontopo) [inline scripts/socialBot.coffee]: don't appear to be using events.

File: scripts/socialBot.coffee [hunk 2]
```diff
@@ -227,7 +247,7 @@ editRSVP = (res) ->
   res.send NEW_DEADLINE(eventName, newDeadline)
 
 test = (res) ->
-  getEvents(res.robot.brain)
+  getEvents(res.robotfo.brain)
```
Comment (lmontopo) [inline scripts/socialBot.coffee]: tyyypo! :P

**Comment by lmontopo** [comment]
Looks good just a couple super quick fixes.

File: scripts/socialBot.coffee (hunk 1 above)
Comment (mahimabhayana) [reply scripts/socialBot.coffee]: Fixed

File: scripts/socialBot.coffee (hunk 2 above)
Comment (mahimabhayana) [reply scripts/socialBot.coffee]: Fixed

**Pull request merged**